doc/users/dflt_style_changes.rst

```diff
@@ -1130,10 +1130,3 @@ mplot3d
   - grid.color
   - grid.linewidth
   - grid.linestyle
-
-
-
-TEMPORARY NOTES TOM IS KEEPING IN THE SOURCE SO THEY DO NOT GET LOST
```

Comment:
I'm just going to merge this as-is, an make a note on 'paper' to do something about this.